src/Api/IriConverterInterface.php

```diff
@@ -44,7 +44,7 @@ public function getItemFromIri(string $iri, bool $fetchData = false);
      *
      * @return string
      */
-    public function getIriFromItem($item, int $referenceType = UrlGeneratorInterface::ABS_PATH) : string;
+    public function getIriFromItem($item, int $referenceType = UrlGeneratorInterface::CONFIG) : string;
```

Comment:
Because otherwise the $referenceType parameter would be set to UrlGeneratorInterface::ABS_PATH and we would have no idea whether we should use the default config value or whether someone wanted to overwrite the default value on the method call.

Another solution would be to remove the default value from the method signature or set it to null.

Do you have another idea?